webapp/channels/src/components/autocomplete_selector.tsx

```diff
@@ -96,7 +109,13 @@ export default class AutocompleteSelector extends React.PureComponent {
             disabled,
             listComponent,
             listPosition,
-        } = this.props;
+        } = {
+            labelClassName: '',
```

Comment:
Why don't you assign the default value for `value` here too? Why do you make it required when it was not before?

Reply:
Made value optional, I set is as required because in `propTypes` it was `PropTypes.string.isRequired`.

Reply:
Oh, I see. Probably the prop types were not well defined :sweat_smile: Sorry about that